File: src/CryptoConseils/BlogBundle/Entity/Image.php
<?php
// src/CryptoConseils/BlogBundle/Entity/Image

namespace CryptoConseils\BlogBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\HttpFoundation\File\UploadedFile;

/**
 * @ORM\Table(name="image")
 * @ORM\Entity(repositoryClass="CryptoConseils\BlogBundle\Repository\ImageRepository")
 * @ORM\HasLifecycleCallbacks
 */
class Image
{
    /**
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @ORM\Column(name="fileExtension", type="string", length=5)
     */
    private $fileExtension;

    /**
     * @var string;
     * @ORM\Column(name="filePath", type="string")
     */
    private $filePath;

    /**
     * @var string
     * @ORM\Column(name="fileName", type="string", nullable=true)
     */
    private $fileName;

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return mixed
     */
    public function getFileExtension()
    {
        return $this->fileExtension;
    }

    /**
     * @param mixed $fileExtension
     */
    public function setFileExtension($fileExtension)
    {
        $this->fileExtension = $fileExtension;
    }

    /**
     * @return string
     */
    public function getFilePath()
    {
        return $this->filePath;
    }

    /**
     * @param string $filePath
     */
    public function setFilePath($filePath)
    {
        $this->filePath = $filePath;
    }

    /**
     * @return string
     */
    public function getFileName()
    {
        return $this->fileName;
    }

    /**
     * @param string $fileName
     */
    public function setFileName($fileName)
    {
        $this->fileName = $fileName;
    }
}

File: app/src/main/java/fr/cryptoconseils/api/model/Image.java
package fr.cryptoconseils.api.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Image {

    @SerializedName("id")
    @Expose
    private Integer id;

    @SerializedName("file_extension")
    @Expose
    private String file_extension;

    @SerializedName("file_path")
    @Expose
    private String file_path;

    @SerializedName("file_name")
    @Expose
    private String file_name;


    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getFileName() {
        return file_name;
    }

    public void setFileName(String url) {
        this.file_name = file_name;
    }


}
